src/foam/nanos/auth/User.js

```diff
@@ -322,6 +322,15 @@ foam.CLASS({
       factory: function() {
         return this.Address.create();
       },
+      label: 'Country',
+      tableCellFormatter: function(value, obj, axiom) {
+        let addressString = '';
+        for ( prop in value.instance_ ) if ( prop ) addressString += ` ${value[prop]}`;
+        return this.__subContext__.countryDAO.find(value.countryId).then((cobj) => {
+          if ( addressString ) this.setAttribute('title', addressString);
```

Comment:
This line doesn't need to be in the callback. It could be above with the 'for'. Also, you could do addressString.trim() to get rid of the extra space at the beginning.